dispatch/unit-tests/old/keysT.cc

```diff
@@ -279,9 +243,6 @@ class keysT : public TestFixture {
         if (debug) cout << __func__ << END << endl;
     }
 
-    /**
```

Comment:
What's the deal with the method comment headers? It helps me find each method... :(

Reply:
Oh, sorry. I thought those were just auto generated and not filled in. Should I add them back in?

Reply:
Wait, that's the keysT.cc code - it's now in 'old' (aka unused).